fastlane/lib/fastlane/documentation/actions_list.rb

```diff
@@ -24,18 +22,18 @@ def self.print_all(platform: nil)
           current << authors.first.green if authors.count == 1
           current << "Multiple".green if authors.count > 1
         else
-          UI.error("Please update your action file #{name} to be a subclass of `Action` by adding ` < Action` after your class name.")
+          UI.error action_subclass_error(name)
           current << "Please update action file".red
+          current << ' '
```

Comment:
Ah, the header references three columns, so I thought it made sense that the row should always have three entries.